src/style.css

```diff
@@ -1,3 +1,82 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+
+  --header-color: #00acdc;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
 body {
+  margin: 0 auto;
+}
+
+header {
+  min-width: 1024px;
+  width: 100%;
+  margin: 0;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+header img {
```

Comment:
It's better to avoid using combined selectors, and using tag selectors to add specific styles (here and for header).
You should have added class for header and for img and add styles using these classes